css/app/calendarlist.css

```diff
@@ -212,6 +212,16 @@
 	box-sizing: border-box;
 }
 
+#app-navigation .app-navigation-list-item {
+	max-height: 1000px;
```

Comment:
Because I need those styles now and not with Nextcloud 11 ;)
But yes, eventually we should unify them in the server and add some developer instructions.